connect.yaml

```diff
@@ -24,9 +24,5 @@ deployAs:
         - key: CTP_CLIENT_SECRET
           description: Commercetools client secret
   - name: enabler
-    applicationType: service
-    endpoint: /
-    configuration:
-      standardConfiguration:
-        - key: CTP_REGION
-          description: Region of Commercetools project
\ No newline at end of file
+    applicationType: assets
+    endpoint: /
```

Comment:
For the `assets` app we can also remove the `endpoint` field as it is not required

Reply:
https://github.com/commercetools/connect-payment-integration-template/commit/7f0656f959c2711fe25576dbd6e1a0314445b6c8

Reply:
Sorry @leungkinghin-ct, missed that commit

Reply:
@abreu-ct no worry :)